syne_tune/experiments/multiobjective.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import List, Tuple, Optional
+
+import pandas as pd
+import numpy as np
+
+from syne_tune.optimizer.schedulers.multiobjective.utils import hypervolume_cumulative
+
+
+DEFAULT_HYPERVOLUME_INDICATOR_NAME = "hypervolume_indicator"
+
+
+def append_hypervolume_indicator(
```

Comment:
I think this is ok for the current solution but please see the comments on alternatives to the trsnsform.